mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/openjpa/AbstractJPAMailboxMessage.java

```diff
@@ -110,12 +113,22 @@
         public MailboxIdUidKey() {
         }
 
-        /** The value for the mailbox field */
+        /**
+         * The value for the mailbox field
+         */
         public long mailbox;
 
-        /** The value for the uid field */
+        /**
+         * The value for the uid field
+         */
         public long uid;
 
+        /**
+         * The value for the attachmentIds field
+         */
+        @ElementCollection(targetClass = String.class)
+        public List<String> attachmentIds;
```

Comment:
I feel there is a concern when placing `attachementIds` inside `MailboxIdUidKey` seems not really related